### ISSUE TITLE
Symmetry groups

### ISSUE THREAD

**Issue by zerothi** [zerothi/sisl#551]
Note that in 3D there exists only 14 different [Bravais lattices](https://en.wikipedia.org/wiki/Bravais_lattice). Maybe they could be introduced for some symmetry-related functionality?

_Originally posted by @tfrederiksen in https://github.com/zerothi/sisl/issues/550#issuecomment-1468894349_
           

Perhaps we should now depend on spglib and use functionality from țhere.

**Comment by tfrederiksen** [comment]
Good idea. Looking at [variables in spglib](https://spglib.readthedocs.io/en/latest/variable.html) their notation is
- `lattice` (basis vectors),
- `position` (atomic points in fractional coordinates), and 
- `types` (Atomic species number).
